pkg/ottl/ottlfuncs/README.md

```diff
@@ -586,6 +587,23 @@ Examples:
 
 - `Concat(["HTTP method is: ", span.attributes["http.method"]], "")`
 
+### Contains
+
+`Contains(target, item)`
+
+The Contains function checks if an item is present in a given slice `target`. It returns true if the `item` is found, and false otherwise.
+
+`target` is a slice of primitive-typed values.
```

Comment:
It works with non-primitive types as well, such as maps, slices, time.Time and all other types described on the OTTL comparison rules.